examples/arguments-parameters-from-configmap.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
```

Comment:
this should result in a failing test, but does not, what is going on?

Reply:
Are you referring to the existence of non-Argo resources in the YAML? I’ve disabled that check for these two examples here: https://github.com/argoproj/argo-workflows/pull/6662/files#diff-692a99fe7ad4eb2b12267c630d5b152dcb5e4eafff4263cca9462afcec3ef53eR10

Reply:
I see. The ConfigMap in the YAML will be ignored since it’s not a Workflow but the current code only print out a warning. 

GetConfigMapValue() returns empty string if a ConfigMap does not exist. I think we should return an error instead. I’ll modify the example to make sure they pass as well.